### PR TITLE
add full metal panic

### PR DESCRIPTION
**Pull request by hollanbm**

### CODE REVIEW

File: series-tmdb.en.yaml [hunk 1]
```diff
@@ -913,6 +913,21 @@ entries:
       - season: 1
         anilist-id: 5114
 
+  - title: "Full Metal Panic!"
+    guid: plex://show/5d9c086fcb3ffa001f1b5059
+    # imdb: https://www.imdb.com/title/tt1355642/
+    # tmdb: https://www.themoviedb.org/tv/31911
+    # tvdb: https://www.thetvdb.com/dereferrer/series/78914
```
Comment (Soitora) [inline series-tmdb.en.yaml:920]: The correct links for all 3 is the following:
```yaml
  # imdb: https://www.imdb.com/title/tt0328739/
  # tmdb: https://www.themoviedb.org/tv/39379
  # tvdb: https://www.thetvdb.com/dereferrer/series/78914
```

File: series-tvdb.en.yaml [hunk 2]
```diff
@@ -961,6 +961,21 @@ entries:
       - season: 1
         anilist-id: 5114
 
+  - title: "Full Metal Panic!"
+    guid: plex://show/5d9c086fcb3ffa001f1b5059
+    # imdb: https://www.imdb.com/title/tt1355642/
+    # tmdb: https://www.themoviedb.org/tv/31911
+    # tvdb: https://www.thetvdb.com/dereferrer/series/78914
```
Comment (Soitora) [inline series-tvdb.en.yaml:968]: The correct links for all 3 is the following:
```yaml
  # imdb: https://www.imdb.com/title/tt0328739/
  # tmdb: https://www.themoviedb.org/tv/39379
  # tvdb: https://www.thetvdb.com/dereferrer/series/78914
```

**Review by Soitora — CHANGES_REQUESTED** [review]

File: series-tvdb.en.yaml (hunk 2 above)
Comment (hollanbm) [reply series-tvdb.en.yaml]: fixed

File: series-tmdb.en.yaml (hunk 1 above)
Comment (hollanbm) [reply series-tmdb.en.yaml]: fixed

**Pull request merged**